2025-05-07 20:54:48,982 INFO:  * Restarting with stat [in C:\restaurant_system\restaurant_system_web\.venv\Lib\site-packages\werkzeug\_internal.py:225]
2025-05-07 20:56:00,176 INFO:  * Restarting with stat [in C:\restaurant_system\restaurant_system_web\.venv\Lib\site-packages\werkzeug\_internal.py:225]
2025-05-07 21:00:44,495 INFO:  * Restarting with stat [in C:\restaurant_system\restaurant_system_web\.venv\Lib\site-packages\werkzeug\_internal.py:225]
2025-05-07 21:03:13,866 INFO:  * Restarting with stat [in C:\restaurant_system\restaurant_system_web\.venv\Lib\site-packages\werkzeug\_internal.py:225]
2025-05-07 21:04:17,158 INFO:  * Restarting with stat [in C:\restaurant_system\restaurant_system_web\.venv\Lib\site-packages\werkzeug\_internal.py:225]
2025-05-07 21:05:28,404 INFO:  * Restarting with stat [in C:\restaurant_system\restaurant_system_web\.venv\Lib\site-packages\werkzeug\_internal.py:225]
2025-05-07 21:06:34,570 INFO:  * Restarting with stat [in C:\restaurant_system\restaurant_system_web\.venv\Lib\site-packages\werkzeug\_internal.py:225]
2025-05-07 21:51:04,472 ERROR: 데이터베이스 초기화 중 오류 발생: Foreign key associated with column 'attendance.employee_id' could not find table 'employee' with which to generate a foreign key to target column 'id' [in C:\restaurant_system\restaurant_system_web\app.py:417]
2025-05-07 21:51:55,972 ERROR: 데이터베이스 초기화 중 오류 발생: Foreign key associated with column 'employee.user_id' could not find table 'user' with which to generate a foreign key to target column 'id' [in C:\restaurant_system\restaurant_system_web\app.py:417]
2025-05-07 21:52:29,745 ERROR: 데이터베이스 초기화 중 오류 발생: Foreign key associated with column 'employee.user_id' could not find table 'user' with which to generate a foreign key to target column 'id' [in C:\restaurant_system\restaurant_system_web\app.py:420]
2025-05-07 21:53:18,172 ERROR: 데이터베이스 초기화 중 오류 발생: Foreign key associated with column 'contracts.employee_id' could not find table 'employees' with which to generate a foreign key to target column 'id' [in C:\restaurant_system\restaurant_system_web\app.py:420]
2025-05-07 21:54:00,917 ERROR: 데이터베이스 초기화 중 오류 발생: Foreign key associated with column 'contracts.employee_id' could not find table 'employees' with which to generate a foreign key to target column 'id' [in C:\restaurant_system\restaurant_system_web\app.py:420]
2025-05-07 21:54:52,302 ERROR: 데이터베이스 초기화 중 오류 발생: Foreign key associated with column 'contracts.employee_id' could not find table 'employees' with which to generate a foreign key to target column 'id' [in C:\restaurant_system\restaurant_system_web\app.py:420]
2025-05-07 22:00:55,518 ERROR: 데이터베이스 초기화 중 오류 발생: Foreign key associated with column 'contracts.employee_id' could not find table 'employees' with which to generate a foreign key to target column 'id' [in C:\restaurant_system\restaurant_system_web\app.py:420]
